app/_hub/kong-inc/pre-function/how-to/_index.md

```diff
@@ -78,33 +69,42 @@ different priority in the plugin chain.
     ...
     ```
 
-6. Test that our Lua code will terminate the request when no header is passed:
+6. Test that the Lua code will terminate the request when no header is passed:
 
     ```bash
     curl -i -X GET http://localhost:8000/test
+    ```
 
+    You should get a `401` response:
+
+    ```
     HTTP/1.1 401 Unauthorized
     ...
     "Invalid Credentials"
     ```
+
     In the logs, there will be the following messages:
```

Comment:
```suggestion
    The logs will show the following messages:
```

In retrospect, you can choose to keep the "Hi there" so you don't have to change this, up to you.